dcp-prototype/frontend/browser/openapi.yml

```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.0
+info:
+  description: |
+    DCP 2.0 Consumer backend API
+  version: "1.0.0"
+  title: DCP 2.0 Consumer 
+servers:
+# Added by API Auto Mocking Plugin
+  - description: SwaggerHub API Auto Mocking
+    url: 'https://api.browser.humancellatlas.org/v1'
+tags:
+  - name: project
+    description: API used to access project information
+  - name: download
+    description: API used to download data
+paths:
+  /projects:
+    get:
+      tags:
+        - project
+      summary: Retrieve filterable information on all projects
+      operationId: getProjects
+      responses:
+        '200':
+          description: OK
+        '400':
```

Comment:
How could you make a bad get request to this endpoint?

Reply:
presently, none